### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by FardaleM**
Small cleanup in the code.

### CODE REVIEW

**Comment by FardaleM** [comment]
Is there anything I can do to have this merge?

File: src/html_util.ml [hunk 1]
```diff
@@ -41,11 +41,12 @@ let html_stub ?(javascript=[]) ~title body_html =
...
-          [css_link ~a:[] ~uri:(make_uri ~service:(static_dir ())
-                                  ["style.css"]) ();
-           css_link ~a:[] ~uri:(make_uri ~service:(static_dir ())
-                                  ["jscalendar"; "calendar-blue2.css"]) ()]))
+       (meta ~a:[ a_name "viewport"; a_content "width=device-width, initial-scale=1" ] () ::
+        css_link ~a:[] ~uri:(make_uri ~service:(static_dir ())
+                                  ["style.css"]) ()::
+        css_link ~a:[] ~uri:(make_uri ~service:(static_dir ())
+                                    ["jscalendar"; "calendar-blue2.css"]) ()::
+        scripts))
```
Comment (glondu) [inline src/html_util.ml:48]: Could you split out this change in a separate commit, please?

Comment (FardaleM) [reply src/html_util.ml:48]: Good catch! I moved the addition of the meta tag to the other PR.
